docs/proposals/resourcebind-life-cycle/resourcebinding-behavior.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Behavior Of ResourceBind When ProPagation Policy ' s ResourceSelector Change
+
+authors:
+- "@olderTaoist"
+
+reviewers:
+- "@XiShanYongYe - Chang"
+- "@weilaaa"
+
+approvers:
+
+creation-date: 2023-06-27
+
+---
+
+# Behavior Of ResourceBind When ProPagation Policy ' s ResourceSelector Change
+
+## Summary
+
+Current there are two situations that `ResourceBinding` will be deleted :
+- the releated resource template be deleted
+- the releated propagation policy be deleted
```

Comment:
Yeah, the summary should describe the current situation for people to get started, but RB won't be deleted by now in case of PropagationPolicy deletion.